cmd/image-builder/main.go

```diff
@@ -252,7 +252,7 @@ func cmdManifestWrapper(pbar progress.ProgressBar, cmd *cobra.Command, args []st
 		if err != nil {
 			return nil, err
 		}
-		img = &imagefilter.Result{distro, archi, imgType}
+		img = &imagefilter.Result{distro, archi, imgType, nil}
```

Comment:
Nitpick: I think at this point it is the right time to change this to multiline form with names where the new field could be simply skipped.